### PR TITLE
fix(curriculum): Add solutions to Bootstrap challenges

### PR DESCRIPTION
**Pull request by Manish-Giri**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Related to #35753

### CODE REVIEW

**Review by RandellDawson — APPROVED** [review]
LGTM

File: curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.english.md [hunk 1]
```diff
@@ -111,7 +111,61 @@ tests:
...
+      <button class="btn btn-block btn-primary">Like</button>
+    </div>
+    <div class="col-xs-4">
+      <button class="btn btn-block btn-info">Info</button>
+    </div>
+    <div class="col-xs-4">
+      <button class="btn btn-block btn-danger">Delete</button>
+    </div>
+  </div>
+  <p>Things cats <span class="text-danger">love</span></p>
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.english.md]:
```suggestion
  <p>Things cats <span class="text-danger">love</span>:</p>
```

File: curriculum/challenges/english/03-front-end-libraries/bootstrap/use-responsive-design-with-bootstrap-fluid-containers.english.md [hunk 2]
```diff
@@ -107,7 +107,62 @@ tests:
...
+  <label><input type="radio" name="indoor-outdoor"> Indoor</label>
+  <label><input type="radio" name="indoor-outdoor"> Outdoor</label>
+  <label><input type="checkbox" name="personality"> Loving</label>
+  <label><input type="checkbox" name="personality"> Lazy</label>
+  <label><input type="checkbox" name="personality"> Crazy</label>
+  <input type="text" placeholder="cat photo URL" required>
+  <button type="submit">Submit</button>
+</form>
+</div>
+
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/03-front-end-libraries/bootstrap/use-responsive-design-with-bootstrap-fluid-containers.english.md]:
```suggestion
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
I only found a couple of typos.  Looks great otherwise.

**Review by moT01 — APPROVED** [review]
This looks good to me. The two changes the @ojeytonwilliams mentioned could be added, then this can be merged. I do not have permission or I would commit them myself, I think it's your repo @Manish-Giri that is blocking me so you will have to do that quick.

**Comment by Manish-Giri** [comment]
@RandellDawson @ojeytonwilliams @moT01 Thanks for the reviews, guys! Sorry, I've been away for a bit due to uni stuff, but I'm back now, so I'll take care of this one tonight.

**Comment by ojeytonwilliams** [comment]
@Manish-Giri Thanks for sorting this out!

**Pull request merged**